store/storetest/team_store.go

```diff
@@ -701,6 +787,36 @@ func testDelete(t *testing.T, ss store.Store) {
 	}
 }
 
+func testPublicTeamCount(t *testing.T, ss store.Store) {
+	o1 := model.Team{}
+	o1.DisplayName = "DisplayName"
+	o1.Name = "z-z-z" + model.NewId() + "b"
+	o1.Email = MakeEmail()
+	o1.Type = model.TEAM_OPEN
+	o1.AllowOpenInvite = true
+	_, err := ss.Team().Save(&o1)
+	require.Nil(t, err)
+
+	teamCount, err := ss.Team().AnalyticsPublicTeamCount()
+	require.Nil(t, err)
+	require.Equal(t, int64(18), teamCount, "should only be 1 team")
+}
+
+func testPrivateTeamCount(t *testing.T, ss store.Store) {
+	o1 := model.Team{}
+	o1.DisplayName = "DisplayName"
+	o1.Name = "z-z-z" + model.NewId() + "b"
+	o1.Email = MakeEmail()
+	o1.Type = model.TEAM_OPEN
+	o1.AllowOpenInvite = false
+	_, err := ss.Team().Save(&o1)
+	require.Nil(t, err)
+
+	teamCount, err := ss.Team().AnalyticsPrivateTeamCount()
```

Comment:
As above here, but the inverse.

Reply:
I'll also add this to the test.  Thanks @grundleborg